### DISCUSSION TITLE
Date Field Change When Update Resource

### DISCUSSION THREAD

**Discussion by ErickZH** [laravel/nova-issues#3540] [Q&A] [answered]
- Laravel Version: 8.58.0
- Nova Version: 3.29.0
- PHP Version: 7.4.21
- Database Driver & Version: MariaDB 10.6.3
- Operating System and Version: Mac OS Mojave 10.14.6
- Browser type and version: Google Chrome 93.0.4577.63
- Reproduction Repository: https://github.com/ErickZH/nova-date-field

### Description:
When I update any field of the resource, the value of the Date field is changed to the current day

**Comment by crynobone** [comment, marked as answer]
I can't reproduce the issue if I change the schema type from `$table->timestamp()` to `$table->date()`. 

I believe the root cause was from MySQL Timestamp default structure:

![image](https://user-images.githubusercontent.com/172966/132303525-790eeceb-000b-4883-b6a9-7caa7ff29152.png)

**Reply by ErickZH** [reply]
Thanks so much @crynobone it work for me! 🚀
